packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -3348,6 +3295,7 @@ class _NavigatorReplaceObservation extends _NavigatorObservation {
   }
 }
 
+<<<<<<< HEAD
```

Comment:
Ah nevermind, here's a merge conflict!  It looks like everything in the "HEAD" block here should be deleted.  It's all stuff from my PR.